Dockerfile

```diff
@@ -0,0 +1,49 @@
+# syntax = docker/dockerfile:1.2  
```

Comment:
I'm not sure if it's tried,

what happens if you build all of them together before creating seperate builders. Turbo can run tasks concurrently so we can leverage that. and if you do "npm i" on base then you can share with others

Reply:
If we run `npm install` before creating separate builders, we won't be able to derive any advantage from using `turbo prune -scope=...`.
However, I have made updates to the Dockerfile to ensure that the image sizes are kept at a minimum - the maximum size being 1.09 GB